src/Process/Installer.php

```diff
@@ -197,11 +192,20 @@ public function getRepoUrl()
             case 'github-https':
                 return "https://github.com/{$this->name}.git";
 
+            case 'gitlab':
+                return "git@gitlab.com:{$this->name}.git";
+                break;
+
             case 'bitbucket':
                 return "git@bitbucket.org:{$this->name}.git";
 
             default:
-                return null;
+                $parts = explode('.', $this->type);
+                if (end($parts) == "git")
+                    return $this->type;
+                else
```

Comment:
There is no need for an else statement since you early returned. Also please use `{}` around the if.